app/jobs/attach_derivative_job.rb

```diff
@@ -0,0 +1,28 @@
+class AttachDerivativeJob < ActiveJob::Base
```

Comment:
This file can come out. It was used for NU's migration from local storage to AWS without re-deriving.